rcswitch/__init__.py

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env python3
 # vim: set encoding=utf-8 tabstop=4 softtabstop=4 shiftwidth=4 expandtab
 #########################################################################
-# Copyright 2017      Daniel Frank                knx-user-forum.de:dafra 
+#  Copyright 2017      Daniel Frank               knx-user-forum.de:dafra 
+#  Copyright 2022      Frank Häfele                  mail@frankhaefele.de
 #########################################################################
-#  This file is part of SmartHomeNG.    https://github.com/smarthomeNG//
+#  This file is part of SmartHomeNG.
+#  https://www.smarthomeNG.de
+#  https://knx-user-forum.de/forum/supportforen/smarthome-py
+#
+#  Sample plugin for new plugins to run with SmartHomeNG version 1.9 and
```

Comment:
Das ist doch kein Beispiel (sample) Plugin oder?

Reply:
erl. in PR für `rcs1000n`